language/English/strings.po

```diff
@@ -16193,9 +16197,14 @@ msgctxt "#37041"
 msgid "Use audio streams that are flagged as "default" (and match the preferred language) regardless of the number of channels or the used codec."
 msgstr ""
 
-#empty strings from id 37042 to 38009
+#. Description of setting "Videos -> Playback -> Configure seek steps"
+#: system/settings/settings.xml
+msgctxt "#37042"
+msgid "Configure seek steps"
```

Comment:
> da-anda commented on xbmc/settings/AdvancedSettings.cpp in 11eeaea 3 hours ago 
>this needs to be more descriptive, but I guess you know and it's a placeholder for now
>xhaggi replied 20 minutes ago
yep, it would be nice if you could write down a nice description i could use for it ;)

;)